sbom/formatted_reader.go

```diff
@@ -53,6 +54,30 @@ func (f *FormattedReader) Read(b []byte) (int, error) {
 			return 0, fmt.Errorf("failed to format sbom: %w", err)
 		}
 
+		// Makes CycloneDX SBOM more reproducible, see
+		// https://github.com/paketo-buildpacks/packit/issues/367 for more details.
+		if f.format.ID() == "cyclonedx-1.3-json" || f.format.ID() == "cyclonedx-1-json" {
```

Comment:
Will `cyclonedx-1-json` always be the latest 1.x version? I'm not super familiar with the format versioning in Syft.

Reply:
Yeah, `cyclonedx-1-json` is [the ID that syft has assigned](https://github.com/anchore/syft/blob/042304ee4c614708b4e293f525c5d0a435ede2cf/internal/formats/cyclonedxjson/format.go#L9) for the CycloneDX output JSON that they generate. This is currently 1.4, but could change in future releases. We have tests that look for 1.4 so if they bump the version, we'll see a failing test to let us know.

Reply:
Great, thank you both for the context